contents/tutorials/cookieless-tracking.md

```diff
@@ -1,91 +1,113 @@
   - product os
 ---
 
-- **Level:** Medium 🦔🦔
-- **Estimated reading time:** 5 minutes ☕️
+Normally, PostHog stores some information about the user in their browser using a cookie. This approach is typical for analytics tools and enables user tracking across sessions, caching feature flag data, and more. 
 
-Normally, PostHog collects information about your users and stores it in a cookie in the users’ browser. This approach is fairly typical and enables you to track users across sessions, but there are some situations where cookie-less tracking is preferable. 
+There are some situations where you don't want to use cookies and do cookieless tracking instead. These include when:
 
-These include:
+- You have concerns about user privacy or regulation such as [GDPR](/docs/integrate/gdpr) or [HIPAA](/docs/privacy/hipaa-compliance).
 
-- If you have concerns about user privacy or regulation such as [GDPR](/docs/integrate/gdpr) or [HIPPA](/docs/privacy/hipaa-compliance).
-- If you have your own system for identifying users across multiple sessions, or if you don’t need to track user identities at all.
+- You have your own system for identifying users across multiple sessions, rely on server-side tracking, or don’t need to track user identities at all.
 
-If you’re interested in trying cookie-less tracking, then this tutorial will explain how to do this by configuring posthog-js to use page memory instead.
+- You hate cookie banners.
+
+This tutorial shows how to configure PostHog's JavaScript Web SDK to do cookieless tracking by using page memory to store user data.
+
+> **Note:** When using cookies, PostHog stores data as a first-party cookie. We don't track users across different sites (like largely blocked [third-party cookies](https://en.wikipedia.org/wiki/Third-party_cookies) do). It also means the same cookie works across subdomains like `posthog.com` and `eu.posthog.com`.
 
 <GDPRForm />
 
 ## Step 1: Decide where to store the data
 
-It can be helpful first to know what data is being stored and why. Specifically, PostHog will usually store the following information in the user’s browser via a cookie:
+It is helpful first to know what data is being stored and why. Specifically, PostHog stores the following information in the user’s browser:
 
-- User ID
+- User `distinct_id`
 - Session ID
-- Device ID 
+- Device ID
 - Active [feature flags](/docs/user-guides/feature-flags)
 - [Super properties](/docs/integrate/client/js#super-properties)
 - Configuration options (e.g., whether [session recording](/docs/user-guides/recordings) is enabled)
 
-As for _why_, this information is tracked only so that PostHog can work optimally and deliver a consistent experience to users.
+If you want to use PostHog without cookies, you must store some of this data elsewhere. Although PostHog has [multiple persistence options](/docs/libraries/js#persistence), the most straightforward is to store it in page memory. This is done by setting `persistance` to `memory`.
```

Comment:
Setting persistance to memory confused me. My immediate questions were "how and where do I do this". Only later I saw its covered in the next step

```suggestion
If you want to use PostHog without cookies, you must store some of this data elsewhere. Although PostHog has [multiple persistence options](/docs/libraries/js#persistence), the most straightforward is to store it in page memory. We show you how to do this in the next step.
```